compiler-rt/lib/sanitizer_common/sanitizer_posix_libcdep.cpp

```diff
@@ -518,6 +522,15 @@ bool IsStateDetached(int state) {
   return state == PTHREAD_CREATE_DETACHED;
 }
 
+bool ShouldTreatRuntimeSecurely() {
+#  if SANITIZER_LINUX
+  // So we can use weak reference from sanitizer_getauxval.h
+  if (&getauxval)
+    return getauxval(/* AT_SECURE */ 23) != 0;
+#  endif
+  return getuid() != geteuid() || getgid() != getegid();
```

Comment:
why not just return false?

Reply:
This is for older versions of glibc that don't have getauxval, or other posix OS that don't have it.